### ISSUE TITLE
Detailed Server Statistics

### ISSUE THREAD

**Issue by ReportCardsMC** [Minehut/Meta#920]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**

Website

**Describe the feature**

Allow server owners to get some more detailed stats that are already tracked in the back-end, such as unique joins, new players in the past x time, total joins in the past x time, average session times, etc. One that I don't think is already completely tracked is favorites, I'd like to see how many players have my server on their favorites list.

I'd also like to see these portrayed in graphs if possible, so it'll be easier to see when you do the best versus when you're at your servers low.

**How the feature is useful**

This feature is useful by providing server owners statistics about their server using data that is already there, just invisible. This feature would be unique over other server hosting platforms, and show the server creator when they are doing the best.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by GeoGb1** [comment]
You can make these things using special plugins or just skript

**Comment by ReportCardsMC** [comment]
> You can make these things using special plugins or just skript

I understand that, but this feature would be nice for any server owner, for vanilla servers, modded servers, etc. It's not as easy to create in vanilla. And you can't track specific data such as favorites, votes, how many newer minehut players join, etc.

**Comment by MrScopes** [comment]
I agree, an analytics tab with average playtime, total joins, and stats alike should definitely exist
